sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -113,6 +113,10 @@ class HadoopTableReader(
 
     val tablePath = hiveTable.getPath
     val inputPathStr = applyFilterIfNeeded(tablePath, filterOpt)
+    val skipHeaderLineCount =
+      tableDesc.getProperties.getProperty("skip.header.line.count", "0").toInt
+    val isTextInputFormatTable =
```

Comment:
Can this be something like `hiveTable.getInputFormatClass.isInstanceOf[TextInputFormat]`?